_dev/apps/ui/src/components/product-feed-page/dashboard/sync-overview.vue

```diff
@@ -107,6 +125,20 @@ export default defineComponent({
     gmcAccountOverviewPage() {
       return getMerchantCenterWebsiteUrls(this.gmcAccountDetails.id).overview;
     },
+    hasProductSelected() {
+      return this.verificationStats.productsInCatalog
+        ? this.verificationStats.productsInCatalog > 0 : null;
```

Comment:
we need confirmation from Sophie to check if we need to display the alert if we haven't configure the product before.